src/EnergyPlus/DataHeatBalance.cc

```diff
@@ -256,6 +256,13 @@ namespace DataHeatBalance {
     int const MixingSourceZonesOnly(1);
     int const AllZones(2);
 
+    // Parameter for zone air flow mass balancing method
+    int const AdjustMixingOnly(1);
+    int const AdjustReturnOnly(2);
+    int const AdjustMixingThenReturn(3);
+    int const AdjustReturnThenMixing(4);
+    int const NoAdjustReturnAndMixing(5);
```

Comment:
Defined five parameters for the different zone air flow balancing enforcing options. The default is NoAdjustReturnAndMixing and will be initialized to "0" instead of "5". I have corrected it and will push the branch later.